### PR TITLE
add docker support

### PR DESCRIPTION
**Pull request by waxzce**
it have to be done with automated build and dockerfile will be in main repository

### CODE REVIEW

**Comment by waxzce** [comment]
Hi, @Geal can you merge it? so I can auto build stuff for docker registry

File: bin/config.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # sozu proxy test config file
 
-command_socket = "./command_folder/sock"
-saved_state    = "./state.json"
+command_socket = "/var/run/sozu/socket"
```
Comment (Geal) [inline bin/config.toml]: please do not modify this test file, make a separate template config for packaging

**Comment by waxzce** [comment]
I reverted change to it, I will provide a better default file later, ATM it does change anything

**Comment by Geal** [comment]
ok. Also, note that the template config file would have to set up the proxy to listen on ports 80 and 443, instead of 8080 and 8443

**Comment by waxzce** [comment]
Thank you, I will provide a better config file for demo later

**Pull request merged**